.github/workflows/unittests.yml

```diff
@@ -18,6 +18,8 @@
 
  jobs:
    unittests:
+     env:
+       RUN_TEST: pytest -v -n 4 spaghetti --cov=spaghetti --doctest-modules --cov-config=.coveragerc --cov-report=xml
```

Comment:
Why 4 threads? GHA gives us 2 afaik. So the difference between 2 and 4 should be minimal.

Reply:
Your are correct, we should go with the `-n auto` option where the free number of cores is chosen. It was before bed time last night when I pushed this up and I think it was a bit drowsy... 😅